annotated_biblios/blameless_postmortems.Rmd

```diff
@@ -0,0 +1,280 @@
+---
+title: "Stats 337, Applied Readings in Data Science"
+output: html_document
```

Comment:
Can you please make this `github_document`, then render and check in the generated markdown?